.github/workflows/build.yaml

```diff
@@ -37,20 +37,19 @@ jobs:
             - '.github/scripts/install_deps.sh'
             - '.github/workflows/build_codecheck.yaml'
           clang_tidy:
-            - 'CMakeLists.txt'
-            - 'cmake/**'
-            - 'src/**'
-            - 'utils/check_clang_tidy_results.py'
-            - 'utils/run-clang-tidy.py'
-            - '.github/workflows/build_clang_tidy.yaml'
-          test_suite:
+            # this job does not compile, but uses cmake, install-dependencies.sh and source files
```

Comment:
Well, if you think  the name is a problem, then how about just "source"?

Reply:
I changed the name.